modules/ROOT/pages/notifications.adoc

```diff
@@ -0,0 +1,19 @@
+= Important Notifications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Use these sources to stay informed about new releases and downtime for Anypoint API Community Manager.
+
+== Release Announcements
+
+Keeping the API Community Manager components updated is critical to keep your API community operating smoothly and optimally. Watch for new releases in these channels so you can plan to xref:update-acm.adoc[update your installation] at your convenience.
+
+New releases are announced at http://trust.mulesoft.com in the "Scheduled Maintenance" section and in xref:release-notes::api-community-manager/api-community-manager-release-notes.adoc[the API Community Manager release notes].
+
+For further information, please contact your MuleSoft customer success representative.
+
+== Downtime
+
+Any downtime of the MuleSoft or Salesforce platforms is tracked and monitored at http://trust.mulesoft.com and http://trust.salesforce.com .
```

Comment:
Omit space
```suggestion
Any downtime of the MuleSoft or Salesforce platforms is tracked and monitored at http://trust.mulesoft.com and http://trust.salesforce.com.
```
Are we saying that support tracks and monitors, or are we advising our audience to track and monitor from these sites?

Reply:
should say "it's communicated" in these sites? - not sure what the best work is; customer need to track and monitor these sites, but we'll publish status updates in them

Reply:
Removed the space and reworded to clarify that Salesforce & MuleSoft track and monitor downtime.